### ISSUE TITLE
Comparison with ncnn?

### ISSUE THREAD

**Issue by crystalpeng** [Tencent/FeatherCNN#6]
Hi, I am new to ncnn and featherCNN, so could you please give me some introdution about the difference between these two framwork? thank you!

**Comment by turbo0628** [comment]
Hey dude, thanks for your interest in FeatherCNN. The two frameworks take different technical paths under the hood, so it's hard to explain in a nut shell. From a user's perspective, both frameworks are aiming at delivering high performance for CNN inference computation. FeatherCNN is very recently released so it may have some problems. It also lacks several layer support for some specific neural networks. Currently ncnn is the more stable project after over a years' development. Performance benchmarks will be provided in the wiki later on, but only for FeatherCNN. As the usage of the two frameworks are very similar, I suggest you to try out both frameworks for your own neural network models, and then pick the one you like.

**Comment by qxin** [comment]
Hi, wonderful work! Thanks. running on NVIDIA TX1( Coretex A57 CPU,) with 10threads. speed : Avg speed, ~85 ms.
./feather_benchmark ./data/mobilenet.feathermodel ./data/input_3x224x224.txt 20 10  (No question but ijust want to  make a small contribution).

**Comment by turbo0628** [comment]
Thank you, but why do you want to launch 10 threads on a TX1? I think there are only 4 CPUs.

**Comment by HolmesShuan** [comment]
We test the run-time speed of `mobilenetv1` on `ncnn` and `FeatherCNN`, based on Rockchip RK3399 (Cortex-A72 1.8GHz x 2 + Cortex-A53 1.5GHz x 4):

Framework (ms) | 1 Thread  | 2 Threads | 3 Threads | 4 Threads
------------ | ------------- | ------------ | ------------- | ----------
Caffe+OpenBLAS* | 250.57 | 204.40 | 248.65 | 230.20
FeatherCNN | 205.76 | 135.17 | **183.34** | **194.67**
NCNN** | **150.95** | **90.79** | 232.31 | 231.64

*: optimized for Cortex-A53 instead of Cortex-A72
**: powersave=0

For a fair comparison, we merge the batch-normalization and scale layers into convolution layers. There is no doubt that `ncnn` has equipped with highly optimized depth-wise convolution operations. `FeatherCNN` is a good work and has great potential.

**Comment by iamhankai** [comment]
horse racing mechanism in Tencent?
